src/Bundle/test/src/Tests/Controller/SubscriptionUiTest.php

```diff
@@ -137,6 +137,23 @@ public function it_allows_deleting_a_subscription(): void
         $this->assertEmpty($subscriptions);
     }
 
+    /** @test */
+    public function it_allows_accepting_a_subscription(): void
```

Comment:
Is this test executed with Symfony state machine? Do we have the same for winzou?